packaged/lisk_snapshot_verify.sh

```diff
@@ -22,9 +22,17 @@
 
 set -eo pipefail
 DB_NAME="lisk_verify"
+
+function clean_up {
+	dropdb $DB_NAME --if-exists
+	rm verify.json
```

Comment:
Shouldn't we make sure the app is stopped also here?

Reply:
doh!

yes 😅

Reply:
Still missing?